app/assets/stylesheets/components/_post.scss

```diff
@@ -80,6 +83,64 @@ $Post-inlineCodeBgColor: rgba(0, 0, 0, 0.04);
     }
   }
 
+  .Table {
```

Comment:
You can, as I just said above, leave line 27 as it is. Saying that it is valid Suit just because it suits you (pun intended) is a stretch though.

I would at least leave a comment there explaining why we did it that way